BackEnd/utils/moderation/handle_approved_images.py

```diff
@@ -0,0 +1,19 @@
+from ..completeness_counter import completeness_count
+
+
+class ApprovedImages:
+    def __init__(self, profile):
+        self.profile = profile
+
+    def delete_approved_image(self, approved_image):
+        if self.profile.status == self.profile.PENDING and approved_image:
+            approved_image.is_deleted = True
+            approved_image.save()
+            completeness_count(self.profile)
+
+    def check_approved_images(self):
+        if self.profile.banner and self.profile.banner.is_deleted:
```

Comment:
Do I get it right, that if `self.profile.banner.is_deleted == True`, we call `delete_approved_image()` method, which (again) sets `is_deleted = True` and saves the instance?